examples/flogo/creditcard/README.md

```diff
@@ -0,0 +1,60 @@
+## Flogo Rules based Creditcard application
+
+
+This example demonstrates rule based processing of credit card application. In this example three tuples are used, tuples description is given below.
+
+
+* `UserAccount` tuple is always stored in network, while the other tuples `NewAccount` and `UpdateCibil` are removed after usage as ttl is given as 0. 
```

Comment:
@LakshmiMekala I feel like using generic terminology like `credit` score instead of `CIBIL` score would be more appropriate.

Reply:
changed from `CIBIL` to `CreaditScore`